MedallionShell/Streams/MergedLinesEnumerable.cs

```diff
@@ -20,6 +26,18 @@ public MergedLinesEnumerable(TextReader standardOutput, TextReader standardError
             this.standardError = standardError;
         }
 
+#if NETSTANDARD2_0_OR_GREATER
+        public IAsyncEnumerator<string> GetAsyncEnumerator(CancellationToken cancellationToken = default)
```

Comment:
- [ ] I suppose we'll need to pass this `cancellationToken` into the `*Internal` method, but how do you use it?

Reply:
Yes, we should pass it to the internal method. The expectation is roughly that any time GetAsyncEnumeratorInternal does a cancelable async operation we should be passing CancellationToken to that operation.